### ISSUE TITLE
pre-processing code

### ISSUE THREAD

**Issue by bingo619** [DarryO/time_lstm#1]
Hi can you please provide the code you used to pre-process the raw data?

and also the train.sh?

**Comment by DarryO** [comment]
Sorry for such a long time before the reply. I have uploaded the preprocess programs. They should be no problem. But they may be not exactly the same with the ones we used before, since it has been a long time...

So if you have any question about the code, please leave me a message.

**Comment by bingo619** [comment]
Thanks.

I have managed to get the data ready for music dataset.

But the training losses for plstm and timelstm1 go to nan after a few epochs.

**Comment by bingo619** [comment]
Thanks.

I have managed to get the data ready for music dataset.

But the training losses for plstm and timelstm1 go to nan after a few epochs.   Timelstm3’s loss won’t decrease.

Can you test?

**Comment by zyody** [comment]
@bingo619 
The preprocess code for lastFM and readme.m file have already been improved. Now you can successfully run the code following the readme.m.

In addition, the new download link for the CiteULike dataset is also provided.
